src/less/icons.less

```diff
@@ -101,6 +106,405 @@
+@cf-icon-settings-round:             "\e629";
+@cf-icon-menu:                       "\e630";
+@cf-icon-menu-round:                 "\e631";
+@cf-icon-lock:                       "\e632";
+@cf-icon-lock-round:                 "\e633";
+@cf-icon-unlock:                     "\e634";
+@cf-icon-unlock-round:               "\e635";
+@cf-icon-clock:                      "\e636";
+@cf-icon-clock-round:                "\e637";
+@cf-icon-chart:                      "\e638";
+@cf-icon-chart-round:                "\e639";
+@cf-icon-play:                       "\e640";
+@cf-icon-play-round:                 "\e641";
+@cf-icon-history:                    "\e642";
+@cf-icon-history-round:              "\e643";
+@cf-icon-table-of-contents:          "\e644";
+@cf-icon-table-of-contents-round:    "\e645";
+@cf-icon-newspaper:                  "\e700";
+@cf-icon-newspaper-round:            "\e701";
+@cf-icon-microphone:                 "\e702";
+@cf-icon-microphone-round:           "\e703";
+@cf-icon-bullhorn:                   "\e704";
+@cf-icon-bullhorn-round:             "\e705";
+@cf-icon-double-quote:               "\e708";
+@cf-icon-double-quote-round:         "\e709";
+@cf-icon-speech-bubble:              "\e710";
+@cf-icon-speech-bubble-round:        "\e711";
+@cf-icon-information:                "\e712";
+@cf-icon-information-round:          "\e713";
+@cf-icon-lightbulb:                  "\e714";
+@cf-icon-lightbulb-round:            "\e715";
+@cf-icon-dialogue:                   "\e716";
+@cf-icon-dialogue-round:             "\e717";
+@cf-icon-date:                       "\e718";
+@cf-icon-date-round:                 "\e719";
+@cf-icon-closing-quote:              "\e720";
+@cf-icon-closing-quote-round:        "\e721";
+@cf-icon-livestream:                 "\e722";
+@cf-icon-livestream-round:           "\e723";
+@cf-icon-parents:                    "\e724";
+@cf-icon-parents-round:              "\e725";
+@cf-icon-servicemembers:             "\e726";
+@cf-icon-servicemembers-round:       "\e727";
+
+
+/* topdoc
+    name: Helper classes for icon sizes
+    family: cf-icons
+    patterns:
+        - name: Large icon size
+          markup: |
+            <span class="cf-icon cf-icon-money cf-icon__lg"></span>
+        - name: 2x icon size
+          markup: |
+            <span class="cf-icon cf-icon-money cf-icon__2x"></span>
+        - name: 3x icon size
+          markup: |
+            <span class="cf-icon cf-icon-money cf-icon__3x"></span>
+        - name: 4x icon size
+          markup: |
+            <span class="cf-icon cf-icon-money cf-icon__4x"></span>
+        - name: 5x icon size
+          markup: |
+            <span class="cf-icon cf-icon-money cf-icon__5x"></span>
+    notes:
+       - "MIT Licensed by Font Awesome"
+*/
+
+// makes the font 33% larger relative to the icon container
+.@{cf-icon-prefix}__lg {
+  font-size: (4em / 3);
+  line-height: (3em / 4);
+  vertical-align: -15%;
+}
+
+.@{cf-icon-prefix}__2x { font-size: 2em; }
```

Comment:
I'm going to double check this. This is how I found it in the Font Awesome source code so that the large icon is somewhat centered, but the 2x, 3x, etc. are at the bottom of the line so developers can overwrite as needed.

I'm going to test this on a new Capital Framework project with to see if the defaults look okay.